packages/openapi-generator/src/generator.ts

```diff
@@ -42,18 +44,20 @@ export async function generate(options: GeneratorOptions): Promise<void> {
   const vdmMapping = readServiceMapping(options);
   const uniqueNameGenerator = new UniqueNameGenerator('-');
   const inputFilePaths = await getInputFilePaths(options.input);
-  inputFilePaths.forEach(async inputFilePath => {
+
+  const promises = inputFilePaths.map(inputFilePath => {
     const uniqueServiceName = uniqueNameGenerator.generateAndSaveUniqueName(
       parseServiceName(inputFilePath)
     );
 
-    await generateFromFile(
+    return generateFromFile(
       inputFilePath,
       options,
       vdmMapping,
       uniqueServiceName
     );
   });
+  return Promise.all(promises).then(() => undefined);
```

Comment:
I still think the code would be a little nicer. The reason is, that a function that has the return type void is a function that returns nothing. In my eyes not returning anything comes more natural than return `undefined`. Consider this example as a non-async case, that we would definitely not implement like this:
```ts
function setSomeGlobalVariable(val: any): void {
  globalVariable = val;
  return undefined;
}
```
```suggestion
  await Promise.all(promises);
```

Reply:
yes agreed. I think your changes does implicitly the same, but to use the feature that if nothing said it is done automatically is more elegant.